inc/namespace.php

```diff
@@ -176,6 +185,7 @@ function enqueue_scripts() {
 						'CognitoRegion' => defined( 'ALTIS_ANALYTICS_COGNITO_REGION' ) ? ALTIS_ANALYTICS_COGNITO_REGION : null,
 						'CognitoEndpoint' => defined( 'ALTIS_ANALYTICS_COGNITO_ENDPOINT' ) ? ALTIS_ANALYTICS_COGNITO_ENDPOINT : null,
 					],
+					'Noop' => (bool) apply_filters( 'altis.analytics.preview', false ),
```

Comment:
Also it's good practice to add docblocks for filters and actions. The cleanest way to do this would be to set the variable on a line above the `wp_enqueue_script()` call like so:

```php
/**
 * The no-op filter allows you to prevent any event tracking or user endpoint
 * updates without needing to update any other code. This is used when previewing
 * posts for example.
 *
 * @param bool $noop If true prevents any analytics events being recorded.
 */
$noop = (bool) apply_filters( 'altis.analytics.noop', false );
```